tools/distpartitioning/dataset_utils.py

```diff
@@ -160,13 +160,95 @@ def get_dataset(input_dir, graph_name, rank, world_size, schema_map):
                 item[1] = node_tids[ntype_name][rank][0]
                 item[2] = node_tids[ntype_name][rank][1]
 
-    #done build node_features locally. 
+    #done building node_features locally. 
     if len(node_features) <= 0:
         logging.info(f'[Rank: {rank}] This dataset does not have any node features')
     else:
         for k, v in node_features.items():
             logging.info(f'[Rank: {rank}] node feature name: {k}, feature data shape: {v.size()}')
 
+    '''
+    Reading edge features now.
+    The structure of the edge_data is as follows, which is present in the input metadata json file. 
+       "edge_data" : {
+            "etype0-name" : {
+                "feat0-name" : {
+                    "format" : {"name": "numpy"},
+                    "data" :   [ #list
+                        "<path>/feat-0.npy",
+                        "<path>/feat-1.npy",
+                        ....
+                        "<path>/feat-<p-1>.npy"
+                    ]
+                },
+                "feat1-name" : {
+                    "format" : {"name": "numpy"}, 
+                    "data" : [ #list 
+                        "<path>/feat-0.npy",
+                        "<path>/feat-1.npy",
+                        ....
+                        "<path>/feat-<p-1>.npy"
+                    ]
+                }
+            }
+       }
+
+    As shown above, the value for the key "edge_data" is a dictionary object, which is 
```

Comment:
Please reformat the comment to make sure it is less than 80.

Reply:
Please ignore this for the time being. Once this PR is merged into the master branch, lets run flake/black on this directory and fix all such issues in a single shot.